static/css/site.css

```diff
@@ -704,9 +704,6 @@ hr{
 		margin:35px 9%;
 		text-align:left;}
 		
-	.banner-head .pure-button {
-		margin:auto;}
-	
```

Comment:
This rule made the buttons stack on top of one another on mobile, removing this makes them split apart and it looks better.